src/Orckestra.Composer.Product/Factory/ProductViewModelFactory.cs

```diff
@@ -687,29 +689,30 @@ protected virtual List<KeyVariantAttributeItem> EnableKvasInStock(GenerateKvaIte
                 .Where(pi => string.Equals(pi.ProductId, productId, StringComparison.InvariantCultureIgnoreCase))
                 .Where(pi => string.Equals(pi.VariantId, variantId, StringComparison.InvariantCultureIgnoreCase))
                 .OrderBy(pi => pi.SequenceNumber)
-                .Select(pi => {
+                .Select(pi =>
+                {
                     var image = ViewModelMapper.MapTo<ProductDetailImageViewModel>(pi, cultureInfo);
                     if (string.IsNullOrEmpty(image.Alt))
                         image.Alt = defaultAlt;
                     return image;
```

Comment:
What if to check whether any image is selected in the first LINQ with productImages? Like 
```
bool hasSelected = false;
...
...
                    var image  = ViewModelMapper.MapTo...
                    if (string.IsNullOrEmpty(image.Alt)) { image.Alt = defaultAlt; }

                    if (image.Selected) { hasSelected = true; }
```
Then we don't have to make a call of Any() with a predicate to iterate and check collection again. And then just
if (!hasSelected)
{
       SetFirstImageSelected(images)
}

Just a suggestion